mk/spksrc.tc-flags.mk

```diff
@@ -16,6 +16,14 @@ ifeq ($(strip $(TC_TYPE)),)
 TC_TYPE = DSM
 endif
 
+ifeq ($(strip $(TC_OS_MIN_VER)),)
+TC_OS_MIN_VER = $(firstword $(subst ., ,$(TC_VERS))).$(firstword $(subst ., ,$(subst $(firstword $(subst ., ,$(TC_VERS))),,$(TC_VERS))))-$(TC_BUILD)
+endif
+
+ifeq ($(strip $(TC_FIRMWARE)),)
+TC_FIRMWARE = $(TC_OS_MIN_VER)
+endif
+
```

Comment:
By adjusting the INFO file generation you may discard TC_FIRMWARE (see my comment)

Reply:
done